### PR TITLE
chore(ci): automatically convert imports to absolute ones 

### PR DESCRIPTION
**Pull request by kszucs**
Alternative version of #3676

### CODE REVIEW

**Comment by jreback** [comment]
+1 on using absolute imports

**Comment by kszucs** [comment]
I tend to prefer absolute imports as well.

**Comment by cpcloud** [comment]
@kszucs Needs a rebase.

**Comment by kszucs** [comment]
Going to rebase after https://github.com/ibis-project/ibis/pull/3276 gets merged.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -28,6 +28,16 @@ repos:
         types:
           - python
         types_or: []
+      - entry: absolufy-imports
+        name: relatify
```
[the remark this answers is not in the document]
Comment (kszucs) [reply .pre-commit-config.yaml]: updated

**Comment by cpcloud** [comment]
@kszucs Can you rebase and fix the merge conflicts? We can merge after that.

**Comment by kszucs** [comment]
@cpcloud rebased

**Comment by cpcloud** [comment]
The duckdb version bump is causing a number of failures. Fixing now.

**Comment by cpcloud** [comment]
One more rebase is necessary 😅

**Comment by kszucs** [comment]
@cpcloud rebased

**Review by cpcloud — APPROVED** [review]

**Pull request merged**